README.md

```diff
@@ -53,7 +53,7 @@ Did you find a typo, a broken link, or another item with an obvious quick fix?
 
 If you can see what the problem is, and you know how to fix it, then you can make a PR (pull request) with the change and contribute to the docs repo yourself.
 
-> Want to make a larger contribution? Please see [CONTRIBUTING.md](./CONTRIBUTING.md) first! 
+> Want to make a larger contribution? Please see [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/README.md) first! 
```

Comment:
This leads to the README. Possibly failed to copy the url?
```suggestion
> Want to make a larger contribution? Please see [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) first! 
```

Reply:
Ohh my God, that's true, Multiple tabs are the culprit here 🤦. Appreciate the suggestion 🧑‍🚀